apps/prairielearn/src/pages/administratorQueries/administratorQueries.ts

```diff
@@ -0,0 +1,60 @@
+import { z } from 'zod';
+import express = require('express');
+const router = express.Router();
```

Comment:
We usually relocate `const router = ...` out of the middle of the `import` statements when refactoring. That is, move this below all the other `import` statements.